src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -1967,7 +1967,14 @@ impl Step for Extended {
     }
 }
 
-fn add_env(builder: &Builder<'_>, cmd: &mut BootstrapCommand, target: TargetSelection) {
+fn add_env(
+    builder: &Builder<'_>,
+    cmd: &mut BootstrapCommand,
+    target: TargetSelection,
+    built_tools: &HashSet<&'static str>,
+) {
+    // envs for wix should be always defined, even if not used
+    // FIXME: is they affect ccache?
```

Comment:
What is this referring to? I don't think we use (s?)ccache for Wix invocations?

It's also not clear to me what we mean by "should always be defined even if not used". Can you elaborate? What happens if we get that wrong?

Reply:
Reference to sccache because this comment from pr with attempt to bump sccache, should be deleted.

As for "defined" - wix fails to build if it can't find all associated with components env vars. i.e. for rustfmt - which is optional, env var `CFG_RUSTFMT` should be always set (with 1 or 0 value), doesn't matter if it rustfmt component build or not. Hmm, need rephrase this.